src/genie/libs/parser/asa/show_route.py

```diff
@@ -30,7 +30,6 @@ class ShowRouteSchema(MetaParser):
                                 Optional('subnet'): str,
                                 'route': str,
                                 Optional('active'): bool,
-                                Optional('date'): str,
```

Comment:
is there a reason the key needs to be removed? if not, can you add it back

Reply:
done. cheers
```text
(genieparser) tests $> python -m unittest
............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................./Users/pedr7260/py/genieparser/src/genie/libs/parser/iosxr/show_controllers.py:78: FutureWarning: Possible nested set at position 74
  p2 = re.compile(r'^mac\=(?P<mac>[A-Fa-f0-9:]+) +vlan=(?P<vlan>\d+)'
/Users/pedr7260/py/genieparser/src/genie/libs/parser/iosxr/show_controllers.py:78: FutureWarning: Possible nested set at position 176
  p2 = re.compile(r'^mac\=(?P<mac>[A-Fa-f0-9:]+) +vlan=(?P<vlan>\d+)'
................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................./Users/pedr7260/py/genieparser/src/genie/libs/parser/utils/common.py:604: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  cmd_node = root.getchildren()[0]
/Users/pedr7260/py/genieparser/src/genie/libs/parser/utils/common.py:617: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  cmd_node = cmd_node.getchildren()
....................................................................../Users/pedr7260/py/genieparser/src/genie/libs/parser/nxos/show_bgp.py:8976: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  for item in nei.getchildren():
/Users/pedr7260/py/genieparser/src/genie/libs/parser/nxos/show_bgp.py:8984: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  root = item.getchildren()[0]
............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 3872 tests in 17.602s

OK
(genieparser) tests - asartsv4 $>
```